modules/sc-mesh-secure-deployment/src/nats/src/comms_command.py

```diff
@@ -260,6 +269,15 @@ def __apply_mission_config(self) -> Tuple[str, str]:
                     ) + str(ret.stdout) + str(ret.stderr)
             self.logger.debug("Mission configurations applied")
             return "OK", "Mission configurations applied"
+            # Start S90mptcp after restarting S9011sNatsMesh
+            ret = subprocess.run(
+                ["/opt/S90mptcp", "start"], shell=False, check=True, capture_output=True, )
+            if ret.returncode != 0:
+                return "FAIL", f"Setting MPTCP configs failed" + str(
+                    ret.returncode
+                ) + str(ret.stdout) + str(ret.stderr)
+            self.logger.debug("Started MPTCP service")
+            return "OK", "Started MPTCP service"
```

Comment:
This code is never executed as there is return "OK" above already

Reply:
Apply command should return --> return "OK", "Mission configurations applied"

Reply:
Changed the return message and also added S90mptcp stop when revoke applied.
https://github.com/tiiuae/mesh_com/commit/401159126efb1a826aaf73f50ac630186877c3ca